pkg/test/framework/features/features.yaml

```diff
@@ -83,6 +83,7 @@ features:
       uninstall_revision:
       uninstall_manifest:
       uninstall_purge:
+      install-revision:
```

Comment:
this doesn't actually test installing a revision.. it tests you can not install a revision